plugins/kubernetes-node/src/extensions.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2023 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { createExtensionPoint } from '@backstage/backend-plugin-api';
+import { KubernetesObjectsProvider } from '@backstage/plugin-kubernetes-backend';
```

Comment:
To give some more context for @benjdlambert's observation -- to avoid a cyclic dependency it probably does make sense to the move the whole `KubernetesObjectsProvider` interface into the `kubernetes-node` plugin.